packages/flutter/lib/src/material/list_tile.dart

```diff
@@ -456,6 +457,12 @@ class ListTile extends StatelessWidget {
   /// Inoperative if [enabled] is false.
   final GestureLongPressCallback? onLongPress;
 
+  /// Handler called when the focus changes.
```

Comment:
Since we're just passing this along to InkWell, best to make InkWell's onFocusChanged API doc an `@template` and refer to it with `@macro` here.